test-result-summary-client/src/Dashboard/Widgets/Graph/ODM.jsx

```diff
@@ -96,12 +95,21 @@ export default class ODM extends Component {
                 if(jdkDate) {
                     resultsByJDKBuild[t.buildName][jdkDate] = resultsByJDKBuild[jdkDate] || [];
                     resultsByJDKBuild[t.buildName][jdkDate].push( {
-                        globalThroughput: t.tests[0].testData.metrics[0].value[0],
+                        globalThroughput: math.round(aggregateInfo.metrics[0].value["mean"], 3),
                         additionalData: {
-                            testId: t.tests[0]._id,
+                            mean: math.round(aggregateInfo.metrics[0].value["mean"], 3),
+                            max: math.round(aggregateInfo.metrics[0].value["max"], 3),
+                            min: math.round(aggregateInfo.metrics[0].value["min"], 3),
+                            median: math.round(aggregateInfo.metrics[0].value["median"], 3),
+                            stddev: math.round(aggregateInfo.metrics[0].value["stddev"], 3),
+                            CI: aggregateInfo.metrics[0].value["CI"],
+                            validIterations: aggregateInfo.metrics[0].value["validIterations"],
+                            testId: (Array.isArray(t.tests) && t.tests.length > 0) ? t.tests[0]._id : null,

```

Comment:
Let's remove `hasChildren `, `testId` and `parentId` since they are not really required.